index.html

```diff
@@ -28,9 +33,12 @@
 
     <!-- headline starts -->
 
-    <section class="headline">
+    <section class="upper-headline">
+      <section class="headline">
         <section class="headline-heading">
-            <h1>I'm Zuheb Ahmed.Glad to see you!</h1>
+            <h1>I'm Zuheb Ahmed.
+              Glad to see you!
+            </h1>
         </section>
         <section class="headline-description">
             <p>
```

Comment:
- [ ] **[OPTIONAL]** Great work on incorporating the changes recommended by the previous reviewer! I have a small suggestion for **your paragraph**. You might want to **add some extra space on the right side** to make it match the layout of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=QfxQv9P39nTqAsxx-0). Additionally, it's worth checking the `line-height` of your `h1` to ensure it looks just right. 🎯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236289786-d630c365-17d3-43b3-bc7a-788ea5478700.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236289433-27d3660a-0179-4aa2-8093-7e3621d70aa2.png)